packages/jaeger-ui/src/components/TracePage/index.test.js

```diff
@@ -699,15 +700,43 @@ describe('<TracePage>', () => {
       expect(track.trackRange.mock.calls).toEqual([[src, range, [0, 1]]]);
     });
   });
+
+  describe('focusSpan', () => {
+    it('calls updateUiFind and focusUiFindMatches', () => {
+      const spanName = 'span1';
+      wrapper.instance().focusSpan(spanName);
```

Comment:
I believe to follow the pattern (generally) present in the code base, this should be: `wrapper.find(TraceTimelineView).prop('focusSpan')(spanName);`

reason being: testing `wrapper.instance().fnName(arg)` can lean towards testing implementation, while testing `someChildComponent.prop('fnName')(arg)` tests functionality (in this case child component is able to set and focus uiFind)